app/js/app-menu.js

```diff
@@ -964,7 +971,133 @@ module.exports = function() {
       var cy = chiseInstance.getCy();
 
       chiseInstance.highlightProcesses(cy.nodes(':selected'));
+      });
```

Comment:
Fix indentation and add a new line after this.

Reply:
Done